SPECS/moby-containerd/Makefile.patch

```diff
@@ -0,0 +1,26 @@
+diff -ru containerd-1.6.6+azure-orig/Makefile containerd-1.6.6+azure/Makefile
+--- containerd-1.6.6+azure-orig/Makefile	2022-01-19 19:13:47.000000000 -0800
++++ containerd-1.6.6+azure/Makefile	2022-02-01 14:39:00.558994210 -0800
+@@ -31,7 +31,7 @@
+ 
+ # Used to populate variables in version package.
+ VERSION ?= $(shell git describe --match 'v[0-9]*' --dirty='.m' --always)
+-REVISION=$(shell git rev-parse HEAD)$(shell if ! git diff --no-ext-diff --quiet --exit-code; then echo .m; fi)
++REVISION ?= $(shell git rev-parse HEAD)$(shell if ! git diff --no-ext-diff --quiet --exit-code; then echo .m; fi)
+ PACKAGE=github.com/containerd/containerd
+ SHIM_CGO_ENABLED ?= 0
+ 
+@@ -255,11 +255,11 @@
+ 
+ man/containerd.8: FORCE
+ 	@echo "$(WHALE) $@"
+-	$(GO) run -mod=readonly ${GO_TAGS} cmd/gen-manpages/main.go $(@F) $(@D)
++	$(GO) run -mod=vendor ${GO_TAGS} cmd/gen-manpages/main.go $(@F) $(@D)
```

Comment:
I guess this was the place that was breaking the whole build :(

Reply:
Might be. I wasn't able to compile with golang 1.17 or 1.18 without this patch (unless I enabled network access via RUN_CHECK=y)